coops-ui/src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -222,18 +224,17 @@ export default {
       routingSlipNumber: null,
       staffPaymentFormValid: false,
       totalFee: 0,
-
       // EntityTypes Enum
       EntityTypes
     }
   },
 
   computed: {
     ...mapState(['currentDate', 'entityType', 'entityName', 'entityIncNo',
-      'entityFoundingDate', 'registeredAddress', 'recordsAddress']),
+      'entityFoundingDate', 'registeredAddress', 'recordsAddress', 'configObject']),
     ...mapGetters(['isRoleStaff']),
 
-    validated () {
+    validated () { 
```

Comment:
Sometimes this causes a lint error. Please check with `npm run lint:nofix`.

Reply:
done